Source/Core/DolphinQt2/GameList/GameListModel.h

```diff
@@ -27,6 +27,7 @@ class GameListModel final : public QAbstractTableModel
 
   // Path of the Game at the specified index.
   QString GetPath(int index) const { return m_games[index]->GetFilePath(); }
+  bool ShouldDisplayGameListItem(const int index);
```

Comment:
const for index is not needed here (in the header). On the other hand, this member function can be const (`bool ShouldDisplay(...) construction;`)